### ISSUE TITLE
Can't control volume on Safari with Voiceover on

### ISSUE THREAD

**Issue by marguinbc** [videojs/video.js#3088]
When using Voiceover on Mac OSX with Safari, you cannot control the volume level.  If you tab over to the volume/mute Voice over reads it as "volume level mute".  Pressing ctrl+option+space on the icon adjusts the volume up or down one level but does not actually toggle the mute/unmute.

Once on the volume/mute control, pressing tab again appears to bring you to the volume slider as it appears highlighted, however there is still a focus box around the volume/mute icon. Pressing ctrl+option+space at this point toggles mute/unmute but does not adjust the volume.

Tabbing to the progress bar and then shift+tab brings you back to the volume control but this time the entire control (icon and slider) appears to have a focus box around it.  Pressing ctrl+option+shift at this point behaves the same as the volume slider.

Tested with v5.7.0 and an iPad 3 with iOS 9.2.x latest.

**Comment by gkatsev** [comment]
This is basically a known issue. If you hit tab from the volume menu button you focused the volume slider and can then change the volume using the arrow keys, ctrl-option-space doesn't make sense for a slider. The way the volume menu button is now, there really isn't much we can do about it to completely fix it for voiceover.
We do have plans to make a fully accessible volume control with @OwenEdwards.

**Comment by OwenEdwards** [comment]
@marguinbc is this Mac OSX, or iOS? You mention an iPad3 with iOS 9.2.x at the end of the issue report, but Voiceover on Mac OSX with Safari at the start.

But @gkatsev is right, this is a known issue with the DOM created by the `volume-menu-button` which places a controllable element (the slider) as a child of another controllable element (the mute button). VoiceOver doesn't like this, but it works okay for sighted users and keyboard-only users.

We'll change the volume control in V6.
